Source/WebCore/page/Page.cpp

```diff
@@ -359,7 +359,7 @@ Page::Page(PageConfiguration&& pageConfiguration)
     , m_alternativeTextClient(WTFMove(pageConfiguration.alternativeTextClient))
     , m_consoleClient(makeUniqueRef<PageConsoleClient>(*this))
 #if ENABLE(REMOTE_INSPECTOR)
-    , m_inspectorDebuggable(makeUniqueRef<PageDebuggable>(*this))
+    , m_inspectorDebuggable(WTFMove(PageDebuggable::create(*this)))
```

Comment:
Here I'm able to get away with keeping it a `WTF::UniqueRef`. I'm wondering if I can do similar in other places involving owning the debuggable to let the code communicate that this is a unique ref. @cdumez

Reply:
Hmm, it turns out `WTFMove` is not needed here.

Reply:
I decided to still use `RefPtr` here instead of the proxies uniquely owning the debuggables—using `WeakPtr`s in `RemoteInspector` and `RemoteConnectionToTarget` still won't be totally correct without also converting the underlying objects to a  `Ref` or `RefPtr` to hold on to them. In that case, we still can't use `unique_ptr` or `UniqueRef` here if we use `Ref`s to the same debuggable objects elsewhere.